src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,50 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class IsoscelesTrapezoid extends Figure implements FindArea {
+    private int biggerBaseSide;
+    private int smallerBaseSide;
+    private double side;
+    private int height;
+
+    public IsoscelesTrapezoid() {
+        super();
+        int firstRandomInt = new Random().nextInt(1, 10);
+        int secondRandomInt = new Random().nextInt(1, 10);
+        if (firstRandomInt > secondRandomInt) {
+            biggerBaseSide = firstRandomInt;
+            smallerBaseSide = secondRandomInt;
+        } else {
+            smallerBaseSide = firstRandomInt;
+            biggerBaseSide = secondRandomInt;
+        }
+        this.height = 10;
+        this.side = getSide();
+    }
+
+    @Override
+    public double getArea() {
+        return (double) (biggerBaseSide + smallerBaseSide) * height / 2;
```

Comment:
The explicit casting of `(biggerBaseSide + smallerBaseSide)` to `double` is unnecessary here, as the multiplication and division operations will automatically promote the result to `double`. However, this does not affect the correctness of the calculation.